dataactcore/models/jobTrackerInterface.py

```diff
@@ -263,37 +263,45 @@ def setJobRowcounts(self, jobId, numRows, numValidRows):
             {"number_of_rows_valid": numValidRows, "number_of_rows": numRows})
         self.session.commit()
 
-    def getSubmissionStatus(self,submission_id):
-        job_ids = self.getJobsBySubmission(submission_id)
+    def getSubmissionStatus(self,submission):
+        job_ids = self.getJobsBySubmission(submission.submission_id)
         status_names = self.getJobStatusNames()
         statuses = dict(zip(status_names,[0]*len(status_names)))
         skip_count = 0
 
         for job_id in job_ids:
             job = self.getJobById(job_id)
-            if job.job_type.name != "external_validation":
+            if job.job_type.name not in ["external_validation", None]:
```

Comment:
There are some jobs that don't have a job_type and their status shouldn't be accounted for since they're not serving a purpose (I can't remember why they're there though)